internal/graph/storagewrapper.go

```diff
@@ -62,7 +61,7 @@ var (
 type iterFunc func(ctx context.Context) (storage.TupleIterator, error)
 
 type CachedDatastore struct {
-	storage.OpenFGADatastore
```

Comment:
So I can compose them

Reply:
Uh? It was already shared across requests as this is not done per request, but at the boot of the service.